workspaces/config/lib/env-replace.js

```diff
@@ -1,9 +1,11 @@
 // replace any ${ENV} values with the appropriate environ.
+// optional "?" modifier can be used like this: ${ENV?} so in case of the variable being not defined, it evaluates into empty string
```

Comment:
```suggestion
// optional "?" modifier can be used like this: ${ENV?} so in case of the variable being not defined, it evaluates into empty string.
```